lib/manager/gomod/artifacts.ts

```diff
@@ -96,7 +96,7 @@ export async function updateArtifacts({
     const vendorDir = join(dirname(goModFileName), 'vendor/');
     const vendorModulesFileName = join(vendorDir, 'modules.txt');
     // istanbul ignore if
-    if (await platform.getFile(vendorModulesFileName)) {
+    if (await readLocalFile(vendorModulesFileName)) {
```

Comment:
I think we should add an `existsLocalFile`function for those checks

Reply:
Yes, I started this but it got even more complicated. I would prefer to optimize it later